### PR TITLE
Fix SHA for go-runner manifest - bump to v0.1.1

### PR DESCRIPTION
**Pull request by dims**
my bad, i used the SHA for amd64 image :(

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-build-image/images.yaml [hunk 1]
```diff
@@ -100,5 +100,5 @@
     - v12.0.1
 - name: go-runner
   dmap:
-    sha256:536ab131b0d0e3b13eb83c985cc0ac9ba7e69e7dac9521e6cacd6a8b6019e0a6:
+    sha256:8ee20934e6c005a9ce8d6d8b7ed23698c3bb80e0b30a3d49e5aeca928cc69bf3:
```
Comment (justaugustus) [inline k8s.gcr.io/images/k8s-staging-build-image/images.yaml:103]: We have to revert this. We shouldn't/can't modify tags in prod.

Comment (dims) [reply k8s.gcr.io/images/k8s-staging-build-image/images.yaml:103]: resolved on slack conversation - https://kubernetes.slack.com/archives/C2C40FMNF/p1588871803210600?thread_ts=1588868307.203500&cid=C2C40FMNF

**Comment by listx** [comment]
This PR was a NOP because the promoter rejects tag moves. We have to either (1) delete prod tag `v0.1.0` from the old digest, or (2) promote with a new tag (e.g., `v0.1.1`). I think (1) is prohibited unless it's an emergency, so that leaves us with (2). @dims

**Comment by dims** [comment]
Ack thanks @listx

**Comment by justaugustus** [comment]
@listx -- Discussing here: https://kubernetes.slack.com/archives/C2C40FMNF/p1588877251218300?thread_ts=1588868307.203500&cid=C2C40FMNF

We're fixing with a `v0.1.1` tag in https://github.com/kubernetes/kubernetes/pull/90852.

**Pull request merged**